platform-sdk/swirlds-jasperdb/src/test/java/com/swirlds/merkledb/files/MockAppender.java

```diff
@@ -14,9 +14,8 @@
  * limitations under the License.
  */
 
-package com.swirlds.common.test.logging;
+package com.swirlds.merkledb.files;
```

Comment:
Is this package correct? it seems unrelated to merkledb

Reply:
I assume it is only used by merkedb tests.